packages/algob/sample-project/js/package.json

```diff
@@ -4,9 +4,11 @@
 	"main": "index.js",
 	"license": "MIT",
 	"devDependencies": {
-		"mocha": "^9.2.0"
+		"@types/mocha": "^9.1.0",
+		"mocha": "^9.1.3",
+		"ts-mocha": "^9.0.2"
```

Comment:
ts-mocha and @types/mocha should be definitely only in `ts` project, not `js`

Reply:
I have only now kept it to ts project. Removed from js.